orchestrate-containers/AnsibleRoles/playbook/host_deploy

```diff
@@ -0,0 +1,9 @@
+#Remember to add "-i host_deploy" in the ansible-playbook command.
+
+[kube-master]
```

Comment:
The INI inventory format is neat when each host has only 1 or 2 variables. YAML is much better for an inventory with a lot of variables. Just fyi, no need to change it.

Reply:
Noted.